apps/web-giddh/src/app/daybook/advance-search/daybook-advance-search.component.html

```diff
@@ -13,6 +13,7 @@ <h3 class="modal-title bg" id="modal-title">Advance Search</h3>
                             <label for="">Date Range</label>
                         </div>
                         <div class="col-sm-9 col-xs-12 custom-datepicker" (click)="showGiddhDatepicker($event)">
+                            <i class="icon-calender calender-addone"></i>
```

Comment:
correct spelling of icon-calender -> **icon-calender**  and calender-addone -> **calendar addon** @shalinee-walkover